### DISCUSSION TITLE
A way to see issues from all repositories I own, grouped

### DISCUSSION THREAD

**Discussion by ericoporto** [community/community#41724] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

With the old dashboard gone, we don't get an easy overview of what's going on in all repositories we own. I can browse and view, and order by date, all issues I created, am mentioned or am assigned. But I can't simply see all issues in repositories I own.

This would be a nice improvement, so I could know what issues are created in my repositories - I have too many repositories and I don't jump on all of them since that would be a lot of work.

**Comment by MitkoVtori** [comment]
### In any page, up on your left, click on issues

![Screenshot 2022-12-14 140041](https://user-images.githubusercontent.com/112943652/207590343-d6eb4f63-0dcb-4beb-8d03-2bb03fd0ae19.png)

**Then, on the search menu add your tags and you're ready :)**

![Screenshot 2022-12-14 140112](https://user-images.githubusercontent.com/112943652/207590422-864ecdef-a246-4150-9a08-683767d364ac.png)

**Reply by ericoporto** [reply]
There's no filter for the ownership of the repositories the issues are in.

**Reply by MitkoVtori** [reply]
> There's no filter for the ownership of the repositories the issues are in.

### There is!
### Just type the author & name of the repo:

![Screenshot 2022-12-14 164147](https://user-images.githubusercontent.com/112943652/207626741-ae46ed76-cf73-419f-b74b-e46c2fbab1f5.png)

**As you can see, those are not one, but two repos :)**

![Screenshot 2022-12-14 164232](https://user-images.githubusercontent.com/112943652/207626952-94f73937-8162-43ba-9db9-d65f076e58e4.png)

**Also, you can add by who's the issue made, is it still open & more**

![Screenshot 2022-12-14 164334](https://user-images.githubusercontent.com/112943652/207626994-47a582f8-37df-4598-914b-a66883422274.png)

**Reply by ericoporto** [reply]
I have like, 200 repositories. I can't type all of them. Is there a wildcard for all repositories of the same owner?

**Reply by MitkoVtori** [reply]
> I have like, 200 repositories. I can't type all of them. Is there a wildcard for all repositories of the same owner?

Unfortunatelly I don't think so, but you can always search on google about it. I think there mait be a code that will show you.
